syntax="proto3";
package pms;

option optimize_for = LITE_RUNTIME;

// this is the msg for transfer msg sequence
message SequenceMsg{
    string      userid      = 1; // userid
    string      msgid       = 2; // message id
    sint64      sequence    = 3; // sequence
}

message PackedSeqnMsg{
    repeated    SequenceMsg     msgs = 1;
}
